packages/component-static-pages/client/pages/AuthorGuide/Full.js

```diff
@@ -62,14 +62,14 @@ const Full = props => (
           reporting. At the Full Submission step, authors should be ready to
           upload a completed version of this form (
           <NativeLink
-            href="https://cdn.elifesciences.org/xpub/guides/transparent_reporting.pdf"
+            href="https://elife-cdn.s3.amazonaws.com/xpub/guides/transparent_reporting.pdf"
```

Comment:
I would use `https://cdn.elifesciences.org/xpub/guides/transparent_reporting.pdf` as this goes through the CDN.